content/issues/2022-05-31-update-centernot-updated.md

```diff
@@ -1,15 +1,18 @@
 ---
 title: Update Center not updated with new released plugins
 date: 2022-05-31T12:00:00-00:00
-resolved: false
-#resolvedWhen: 2022-05-30T11:00:00-00:00
+resolved: true
+resolvedWhen: 2022-06-01T08:10:00-00:00
 # Possible severity levels: down, disrupted, notice
 severity: disrupted
 affected:
   - updates.jenkins.io
 section: issue
 ---
 
+[Close Issue]
+the new certificate had been issued and stored within the trusted controller. Both the crawler and the update-center jobs run smoothly again.
```

Comment:
```suggestion
the new certificate had been issued and stored within the trusted controller. Both the crawler and the update-center jobs run smoothly again since 07:34am UTC.

The plugin "gap" was filled at 09:47am UTC as per <https://github.com/jenkins-infra/helpdesk/issues/2950#issuecomment-1143374848>.


```